browser-interface/packages/shared/dao/sagas.ts

```diff
@@ -216,6 +262,15 @@ function* cacheCatalystRealm() {
   const dao: string = yield select((state) => state.dao)
   const realmAdapter: IRealmAdapter = yield call(waitForRealm)
 
+  if (
+    isOnboarding &&
+    realmAdapter.about.configurations?.realmName !==
+      getFeatureFlagVariantValue(store.getState(), 'new_tutorial_variant')
+  ) {
+    store.dispatch(saveProfileDelta({ tutorialStep: 256 }))
```

Comment:
using store.dispatch inside a saga is forbidden and breaks the code, use `yield put(saveProfileDelta({ tutorialStep: 256 }))` instead
```suggestion
    yield put(saveProfileDelta({ tutorialStep: 256 }))
```